ws/sha1.ts

```diff
@@ -9,31 +9,34 @@
 /*jslint bitwise: true */
 
 const HEX_CHARS = "0123456789abcdef".split("");
-const EXTRA = [-2147483648, 8388608, 32768, 128];
-const SHIFT = [24, 16, 8, 0];
+const EXTRA = Int32Array.of(-2147483648, 8388608, 32768, 128);
+const SHIFT = Uint32Array.of(24, 16, 8, 0);
 
 const blocks = [];
 
 export class Sha1 {
-  blocks;
-  block;
-  start;
-  bytes;
-  hBytes;
-  finalized;
-  hashed;
-  first;
-
-  h0 = 0x67452301;
-  h1 = 0xefcdab89;
-  h2 = 0x98badcfe;
-  h3 = 0x10325476;
-  h4 = 0xc3d2e1f0;
-  lastByteIndex = 0;
-
-  constructor(sharedMemory: boolean = false) {
+  private blocks: number[];
```

Comment:
We tend to follow the convention for privates of having a leading `_` sigil, so `_blocks`.  In theory this will make refactoring easier when the TC39 proposal for private members is finally delivered as part of the standard.  (We will be able to find/replace `_foo` for `#foo` and `this._foo` for `this.#foo`.

Reply:
I thought all deno's code database use only in typescript environment. In this case adding `_` prefix for private members looks unnecessary because ts diagnostics always throw error during compilation and language server never pretend to choose private members outside of existing class context.

Reply:
While that is true, that it is enforced, there is still a risk of downstream name collision of something you want public... if you provide a sigil `_` even if it conflicts downstream (and TypeScript disallows this) you don't _care_ because it is private.

The main point though is as I mentioned above, when private members become part of the language standard, in most cases we will want to refactor to make them truly private, and it will be infinitely easier if they follow a particular pattern.  It has been a convention for a long time in JavaScript to preface with a `_` sigil.  Private members will have the `#` sigil, and this needs to change both in the declaration and the call site.  In TypeScript the `private` keyword is only at the declaration site, which would make identifying the callsite more difficult.  A `_` sigil solves that.